apps/web/src/components/credit-cards/credit-card.tsx

```diff
@@ -6,21 +6,22 @@ import css from './card.module.css'
 import AppLink from '@/components/app-link/app-link'
 import Heading from '@/components/heading'
 
-export interface CardProps {
+export interface CreditCardProps {
   helpText: string
   href: string | { pathname: string; query?: { [key: string]: string } }
   icon: ReactNode
   isDisabled: boolean
   title: string
 }
 
-export default function Card({
+/** A credit card */
+export default function CreditCard({
```

Comment:
I spent some time messing around with this component before realizing it wasn't a "card" in the sense that some UI libraries use.

Reply:
Totally cool with renaming and/or reworking the components, but this component isn't specific to credit cards. This is used for the methods on the checkout page, which will include credit cards, wire payments, and crypto payments if all are enabled.

Reply:
Ahhh crud. That makes sense though! Ok I'll have to take a closer look at this. I don't think it makes sense to have a global card components that doesn't accept `children`. Do you think it would make sense to move it into `components/purchase-form`? Or actually, looking at this now, maybe we just rename it to something like `PaymentOptionItem` and `PaymentOptions`. Thoughts?